### PR TITLE
INTERNAL - Add commerce as product dependency

### PR DESCRIPTION
**Pull request by rojoangel**
## Description

Add commerce as a product dependency

## Related Issue

N/A

## Motivation and Context

This marks starter kit projects as dependent on Commerce

## How Has This Been Tested?

N/A
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by pru55e11 — COMMENTED** [review]
Do you need to require any specific commerce version? This will be displayed int he agreement that the customer accepts during aquisition

**Comment by rojoangel** [comment]
As far as I am concerned, no specific version is needed, but let me double-check internally to play on the safe side.

**Comment by rojoangel** [comment]
On second thought, since the kit relies on the `Adobe I/O Events for Adobe Commerce` which requires Adobe Commerce 2.4.4 or higher, I think it may make sense to set this min version.

**Review by pru55e11 — APPROVED** [review]
LGTM!

**Review by pru55e11 — CHANGES_REQUESTED** [review]
actually, I think COMMC will be the code needed.

**Review by nejo — APPROVED** [review]

File: app.config.yaml [hunk 1]
```diff
@@ -54,3 +54,6 @@ application:
         license: Apache-2.0
         actions:
           $include: ./actions/stock/external/actions.config.yaml
+productDependencies:
```
Comment (pablomoreno61) [inline app.config.yaml:57]: Didn't know about this! I just found a wiki mentioning something about it. Do you have more info about possible codes and the impact of this addition in an app builder project?

**Review by pablomoreno61 — APPROVED** [review]

**Pull request merged**